test-integration/utils/actions/atmGovernance.js

```diff
@@ -0,0 +1,180 @@
+const BigNumber = require("bignumber.js");
+const { atmGovernance: atmGovernanceEvents } = require("../../../test/utils/events");
+
+const addGeneralSetting = async (
```

Comment:
I think you are not exporting the functions at the end:

```js
module.exports = {
   addGeneralSetting,
   ...,
}
```

Reply:
adding in